oap-server/server-storage-plugin/storage-doris-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/doris/dao/DorisStorageDAO.java

```diff
@@ -0,0 +1,207 @@
+        LOGGER.warn("newMetadataQueryDAO not implemented yet, returning null.");
+        return null;
+    }
+
+    public IAggregationQueryDAO newAggregationQueryDAO() {
+        LOGGER.warn("newAggregationQueryDAO not implemented yet, returning null.");
+        return null;
+    }
+
+    public IAlarmQueryDAO newAlarmQueryDAO() {
+        LOGGER.warn("newAlarmQueryDAO not implemented yet, returning null.");
+        return null;
+    }
+
+    public ITopologyQueryDAO newTopologyQueryDAO() {
+        LOGGER.warn("newTopologyQueryDAO not implemented yet, returning null.");
+        return null;
+    }
+
+    public IProfileTaskQueryDAO newProfileTaskQueryDAO() {
+        LOGGER.warn("newProfileTaskQueryDAO not implemented yet, returning null.");
+        return null;
+    }
+
+    public IProfileTaskLogQueryDAO newProfileTaskLogQueryDAO() {
+        LOGGER.warn("newProfileTaskLogQueryDAO not implemented yet, returning null.");
+        return null;
+    }
+    
+    public IProfileThreadSnapshotQueryDAO newProfileThreadSnapshotQueryDAO() {
+        LOGGER.warn("newProfileThreadSnapshotQueryDAO not implemented yet, returning null.");
+        return null;
+    }
+
+    public UITemplateManagementDAO newUITemplateManagementDAO() {
+        LOGGER.warn("newUITemplateManagementDAO not implemented yet, returning null.");
+        return null;
+    }
+    
+    public UIMenuManagementDAO newUIMenuManagementDAO() {
+        LOGGER.warn("newUIMenuManagementDAO not implemented yet, returning null.");
+        return null;
+    }
+
+    public IEBPFProfilingTaskDAO newEBPFProfilingTaskDAO() {
+        LOGGER.warn("newEBPFProfilingTaskDAO not implemented yet, returning null.");
+        return null;
+    }
+
+    public IEBPFProfilingScheduleDAO newEBPFProfilingScheduleDAO() {
+        LOGGER.warn("newEBPFProfilingScheduleDAO not implemented yet, returning null.");
+        return null;
+    }
+
+    public IEBPFProfilingDataDAO newEBPFProfilingDataDAO() {
+        LOGGER.warn("newEBPFProfilingDataDAO not implemented yet, returning null.");
+        return null;
+    }
+
+    public IEventQueryDAO newEventQueryDAO() {
+        LOGGER.warn("newEventQueryDAO not implemented yet, returning null.");
+        return null;
+    }
+    
+    public IBrowserLogQueryDAO newBrowserLogQueryDAO() {
+         LOGGER.warn("newBrowserLogQueryDAO not implemented yet, returning null.");
+        return null;
+    }
+
+    public ISpanAttachedEventQueryDAO newSpanAttachedEventQueryDAO() {
+        LOGGER.warn("newSpanAttachedEventQueryDAO not implemented yet, returning null.");
+        return null;
+    }
+
+    public IZipkinQueryDAO newZipkinQueryDAO() {
+        LOGGER.warn("newZipkinQueryDAO not implemented yet, returning null.");
+        return null;
+    }
+
+    public ITagAutoCompleteQueryDAO newTagAutoCompleteQueryDAO() {
+        LOGGER.warn("newTagAutoCompleteQueryDAO not implemented yet, returning null.");
+        return null;
+    }
```

Comment:
All these are returning NULL, so this PR is not completely ready. I am going to move this as DRAFT, and wait for your update.